src/pages/RFOX/components/Claim/ClaimSelect.tsx

```diff
@@ -0,0 +1,121 @@
+
+type ClaimRowProps = {
+  assetId: AssetId
+  amount: string
+  status: string
+  setClaimQuote: (quote: RfoxClaimQuote) => void
+}
+
+const hoverProps = { bg: 'gray.700' }
+
+const ClaimRow: FC<ClaimRowProps> = ({ assetId, amount, status, setClaimQuote }) => {
+  const translate = useTranslate()
+  const history = useHistory()
+
+  const claimAsset = useAppSelector(state => selectAssetById(state, assetId))
+  const assetSymbol = claimAsset?.symbol
+  const claimAmountCryptoBaseUnit = toBaseUnit(bnOrZero(amount), claimAsset?.precision ?? 0)
+
+  // TODO(apotheosis): make this programmatic when we implement multi-account
+  const claimAssetAccountId = useAppSelector(state =>
+    selectFirstAccountIdByChainId(state, claimAsset?.chainId ?? ''),
+  )
+
+  const claimQuote: RfoxClaimQuote = useMemo(
+    () => ({
+      claimAssetAccountId: claimAssetAccountId ?? '',
+      claimAssetId: assetId,
+      claimAmountCryptoBaseUnit,
+    }),
+    [assetId, claimAmountCryptoBaseUnit, claimAssetAccountId],
+  )
+
+  const handleClick = useCallback(() => {
+    setClaimQuote(claimQuote)
+    history.push(ClaimRoutePaths.Confirm)
+  }, [claimQuote, history, setClaimQuote])
+
+  return (
+    <Flex
+      as={Button}
+      align='center'
+      variant='unstyled'
+      p={8}
+      borderRadius='md'
+      width='100%'
+      onClick={handleClick}
+      _hover={hoverProps}
+    >
+      <Box mr={4}>
+        <AssetIconWithBadge assetId={foxAssetId}>
+          <TransactionTypeIcon type={TransferType.Receive} />
+        </AssetIconWithBadge>
+      </Box>
+      <Box mr={4}>
+        <RawText fontSize='sm' color='gray.400' align={'start'}>
+          {translate('RFOX.unstakeFrom', { assetSymbol })}
+        </RawText>
+        <RawText fontSize='xl' fontWeight='bold' color='white' align={'start'}>
+          {assetSymbol}
+        </RawText>
+      </Box>
+      <Box flex='1' alignItems={'end'}>
+        <RawText fontSize='sm' fontWeight='bold' color='green.300' align={'end'}>
+          {status}
+        </RawText>
+        <RawText fontSize='xl' fontWeight='bold' color='white' align={'end'}>
+          <Amount.Crypto value={amount} symbol={assetSymbol ?? ''} />
+        </RawText>
+      </Box>
+    </Flex>
+  )
+}
+
+type ClaimSelectProps = {
+  setClaimQuote: (quote: RfoxClaimQuote) => void
+}
+
+export const ClaimSelect: FC<ClaimSelectProps & ClaimRouteProps> = ({
+  headerComponent,
+  setClaimQuote,
+}) => {
+  return (
+    <SlideTransition>
+      <Stack>{headerComponent}</Stack>
+      <CardBody py={12}>
+        <Flex flexDir='column' gap={4}>
+          <ClaimRow
+            assetId={foxAssetId}
+            amount={'1500'}
+            status={'Available'}
+            setClaimQuote={setClaimQuote}
+          />
+          <ClaimRow
+            assetId={foxAssetId}
+            amount={'200'}
+            status={'Available'}
+            setClaimQuote={setClaimQuote}
+          />
```

Comment:
Dummy rows for now.

Reply:
Note, we'll also want an empty state i.e 

<img width="455" alt="image" src="https://github.com/shapeshift/web/assets/17035424/f6c19167-6272-426f-9055-2038a9f5b5a5">

Obviously doesn't have to be mock-"wired up" for now since dummy state means we either go with showing rows or empty state, but would've expected to see the empty state present in this PR regardless 🙏🏽

Reply:
Right you are, ser! Good catch, I'll add that in 🙏

Reply:
Added in https://github.com/shapeshift/web/pull/7004/commits/7f9a0c5a6cabd358dfe28feea44261801ebeb451